docs/source/_includes/create-genesis-block.inc

```diff
@@ -73,7 +73,7 @@ genesis block also includes the keys for the other nodes in the initial network.
         -o config-consensus.batch \
         sawtooth.consensus.algorithm.name=pbft \
         sawtooth.consensus.algorithm.version=VERSION \
-        sawtooth.consensus.pbft.members=[VAL1KEY,VAL2KEY,...,VALnKEY]
+        sawtooth.consensus.pbft.members='["VAL1KEY","VAL2KEY",...,"VALnKEY"]'
```

Comment:
Please also change the VALxKEY stuff in the following note to:

> Replace ``'["VAL1KEY,","VAL2KEY","VAL3KEY","...,"VALnKEY"]'`` with the ...

And add a sentence to the end of that paragraph:

> Be sure to use single quotes and double quotes correctly, as shown in the example.

Reply:
I have added this sentence to the note. I also added this sentence to the tip in docs/source/sysadmin_guide/testing_sawtooth.rst